index.css

```diff
@@ -0,0 +1,492 @@
+  display: grid;
+  grid-template-columns: repeat(2, minmax(0, 1fr));
+}
+
+.card {
+  background-color: #fff;
+  padding: 16px 16px 40px;
+  border-radius: 16px;
+}
+
+.p-highlight {
+  list-style: none;
+  padding-inline-start: 0;
+  margin-top: 0;
+}
+
+.p-highlight li::before {
+  content: '\2022';
+  color: #bcc0c9;
+  font-weight: bold;
+  font-size: 1.5rem;
+  display: inline-flex;
+  width: 1rem;
+  margin-left: -1rem;
+  margin-right: 0.3rem;
+  vertical-align: sub;
+}
+
+.p-highlight li:first-child::before {
+  color: transparent;
+  margin-right: 0;
+  text-transform: uppercase;
+}
+
+.p-highlight li {
+  font-size: 13px;
+  color: #7a869a;
+}
+
+.skill-set > ul > li > img {
+  width: 48px;
+  height: 48px;
+}
+
+.skill-set > ul > li > span {
+  font-size: 15px;
+  font-weight: 500;
+  line-height: 20px;
+}
+
+.skill-set > ul {
+  height: 248px;
+  margin: 0 0 28px;
+  padding: 12px;
+}
+
+.skill-set > ul > li {
+  gap: 16px;
+  margin: 6px 0;
+  padding: 9px 12px;
+  border-radius: 8px;
+  flex: 1;
+  color: #253858;
+  background-color: #f7f7f9;
+}
+
+.p-highlight li:first-child {
+  text-transform: uppercase;
+  font-weight: 700;
+  color: #344563;
+}
+
+.list-tags > * {
+  background-color: #ebebff;
+  padding: 4px 12px;
+  list-style: none;
+  color: #6070ff;
+  font-size: 12px;
+  border-radius: 10px;
+}
+
+.connect {
+  color: #7f8cff;
+  margin-top: 10px;
+}
+
+.connect .social li {
+  list-style: none;
+  cursor: pointer;
+}
+
+.social > li > a > img {
+  max-width: 20px;
+  height: 19px;
+}
+
+.connect .social {
+  gap: 10px;
+}
+
+.image-auto-h {
+  width: 100%;
+  height: auto;
+}
+
+.about-me {
+  background-color: #fff;
+  padding: 108px 24px;
+  flex-wrap: nowrap;
+  border-radius: 0 50px 0 0;
+}
+
+.about-me > div {
+  flex: 1;
+}
+
+.btn-action {
+  padding: 12px 16px;
+  border-radius: 8px;
+  text-align: center;
+  font-size: 17px;
+  color: #396df2;
+  border: 1px solid #396df2;
+  background-color: #fff;
+}
+
+.btn-action:hover {
+  color: #fff;
+  background: #6070ff;
+}
+
+.btn-action:active {
+  background-color: #2230d2;
+}
+
+.tech {
+  height: 28px;
+  padding: 20px 0;
+  margin-left: 12px;
+  border-bottom: solid 1px #acb7c3;
+}
+
+.lang {
+  border-bottom: none;
+  margin-top: 35px;
+  margin-bottom: 12px;
+}
+
+.tech h4 {
+  font-size: 20px;
+  font-weight: 500;
+}
+
+.round-border-8 {
+  border-radius: 10px;
+}
+
+hr {
+  color: #dfe1e6;
+}
+
+.tech > button {
+  color: #acb7c3;
+  border: none;
+  background-color: #fff;
+  padding: 0;
+  margin-right: 42px;
+  cursor: pointer;
+}
+
+h6 {
+  font-weight: 500;
+  font-size: 20px;
+}
+
+.about-contact {
+  margin-top: 28px;
+  background-color: #fff;
+}
+
+.contact-me {
+  background: url(./images/contact-bg.png);
+  background-size: 70% auto;
+  background-repeat: no-repeat;
+  background-color: #6070ff;
+  border-radius: 50px 0 0 0;
+  background-position: top 1% right 0%;
+}
+
+.contact-container {
+  padding: 60px 16px 45px;
+  color: #fff;
+}
```

Comment:
- Kindly, you are giving unnecessary padding between the title and detail, and also your placeholder is very small, please update it to have a better design :+1: 

your design
![Screenshot from 2023-03-01 22-44-10](https://user-images.githubusercontent.com/52010789/222249903-bbb3d56b-1571-4ec0-aae2-4f9280a66849.png)
Figma
![Screenshot from 2023-03-01 22-44-10](https://user-images.githubusercontent.com/52010789/222250092-eaa7ebe1-2aca-4a60-9ef4-418c64986ecc.png)
 design

Reply:
@binhussen 
Hello Sir. I suggest you double-check next time the screenshots, please. You accidentally sent the same one twice which was confusing at first 😅. But you are correct, we will fix it right away. Thank you for your time and review. You rock!
![RockRockAndRollGIF](https://user-images.githubusercontent.com/33934969/222256374-bf334852-41b5-49d2-93a2-74ea1fe121e1.gif)